Removing the event banner off the home page

clcoffey opened the pull request:
- I commented this out as we will want to bring it back for our December community call.

lenucksi reviewed (approved):
LGTM

claredillon commented:
@lenucksi - not sure what's happening here - looks like changes above merged - but the new images not showing up on website: http://innersourcecommons.org/events/isc-fall-2020/

Any ideas?

lenucksi commented:
> @lenucksi - not sure what's happening here - looks like changes above merged - but the new images not showing up on website: http://innersourcecommons.org/events/isc-fall-2020/
> 
> Any ideas?

The build failed, apparently due to some type / file not found / ... issue, take a look here, right at the bottom of all logs: https://travis-ci.org/github/InnerSourceCommons/innersourcecommons.org/builds/721331006

dsugrobov commented:
@claredillon you should specify the proper format of the picture.
`png` file in the [folder](https://github.com/claredillon/innersourcecommons.org/tree/patch-25/assets/img) and `jpg` on the page:

```
internal image /assets/img/Martin_Woodward.jpg does not exist (line 516)
```

lenucksi commented:
> @claredillon you should specify the proper format of the picture.
> `png` file in the [folder](https://github.com/claredillon/innersourcecommons.org/tree/patch-25/assets/img) and `jpg` on the page:
> 
> ```
> internal image /assets/img/Martin_Woodward.jpg does not exist (line 516)
> ```

Also: We should switch over to GitHub Actions such that the build logs are easier to find for everyone including @claredillon making self-help easier :smile:

clcoffey committed:
removing the event banner off the home page

- I commented this out as we will want to bring it back for our December community call.
